agir/front/components/genericComponents/Card.js

```diff
@@ -53,10 +50,14 @@ const Card = styled.div`
         : ""}
   }
 
+  box-shadow: ${style.cardShadow};
+  border-radius: 8px;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```